packages/cli-lib/lib/lang.js

```diff
@@ -0,0 +1,75 @@
+const util = require('util');
+const path = require('path');
+const fs = require('fs-extra');
+const handlebars = require('handlebars');
+const yaml = require('js-yaml');
+const { logger } = require('../logger');
+
+const MISSING_LANGUAGE_DATA_PREFIX = '[Missing language data]';
+
+let languageObj;
+
+const loadLanguageFromYaml = () => {
+  if (languageObj) return;
+
+  try {
+    languageObj = yaml.load(
+      fs.readFileSync(path.join(__dirname, '../lang/en.lyaml'), 'utf8')
+    );
+    logger.debug(
+      'Loaded language data: ',
+      util.inspect(languageObj, true, 999, true)
+    );
+  } catch (e) {
+    logger.error('Error loading language data: ', e);
+  }
+};
+
+const getTextValue = lookupDotNotation => {
+  const lookupProps = lookupDotNotation.split('.');
+  const missingTextData = `${MISSING_LANGUAGE_DATA_PREFIX}: ${lookupDotNotation}`;
+  let textValue = languageObj;
+
+  try {
+    lookupProps.forEach(prop => {
+      textValue = textValue[prop];
+    });
+  } catch (e) {
+    logger.error(`Unable to access language property: ${lookupDotNotation}`);
```

Comment:
Could we be more specific here about which prop we failed on?

Reply:
Good idea. I'll add this.